### DISCUSSION TITLE
I imagine Devs must hate that question ...

### DISCUSSION THREAD

**Discussion by ofmarconi** [wpengine/faustjs#236] [Q&A] [answered]
Is there an effort to make it compatible with elementor?

I know that it may not be easy, but it is so fast to build on it, if even with some limitations, this framework of yours would work on it, it would be incredible.

Which builders are working with this framework?

Would it be possible one day, everything we have in the wp to be compatible with this framework?

Thanks! and congratulations!

elementor errors: https://i.postimg.cc/52H3V6Q6/Captura-de-Tela-2021-05-20-a-s-13-21-43.png

**Comment by josephfusco** [comment, marked as answer]
Hey @ofmarconi we apologize for the late reply. Support for Elementor is not currently on our roadmap however we may consider this for the future.
